src/components/Routes/Routes.js

```diff
@@ -19,6 +20,16 @@ export const paths = {
 };
 
 export function Routes() {
+  const sendAnalytics = useSendAnalytics();
+  let location = useLocation();
+
+  useEffect(() => {
+    // Send an analytics event when the URL changes.
+    sendAnalytics({
+      path: window.location.href.replace(window.location.origin, "")
```

Comment:
Above location isn't used, did you mean to use it here instead of `window.location`?

Reply:
This was copied over from the history listener in App.js. I presume it was done this way because you want the analytics to get the full location, not the location that the router returns (which from memory excludes the basename).

Reply:
Ahh yes of course, thanks for reminding me! :)